app/controlplane/api/workflowcontract/v1/crafting_schema.proto

```diff
@@ -91,6 +91,9 @@ message CraftingSchema {
 
       // Chainloop attestation coming from a different workflow.
       ATTESTATION = 12;
+      CSAF_INFORMATIONAL_ADVISORY = 13;
```

Comment:
Great! nitpick, Just for visiblity, could you bring closer `CSAF_VEX = 8;` so they are all grouped :)

Reply:
Sure thing! :)